CHANGELOG.rst

```diff
@@ -7,6 +7,12 @@ Change Log
 ----------
 
 
+7.9.0
```

Comment:
I think a major version 8.0.0 would be wise, so that if there's an issue, it's easy for someone to use `^7.x.` kinds of notation to select stuff that doesn't have this until a fix is ready. Probably not an issue, but that thing about change in required arg worries me.

Reply:
@willronchetti do you have an opinion on this?

Reply:
I'm a little worried about that required arg. I note that if you [search for calls to yaml.load in 4dn-dcic](https://github.com/search?q=org%3A4dn-dcic%20yaml.load&type=code) (which is only one of our 3 orgs), you'll see several calls that don't pass an explicit Loader argument. Will that work or are you forcing code updates? Maybe those updates are fine. If there is an obvious loader, maybe we should have something that someone can call instead.

Reply:
Yeah, looking at those results as well as for [dbmi-bgm](https://github.com/search?q=org%3Adbmi-bgm%20yaml.load&type=code) and [smaht-dac](https://github.com/search?q=org%3Asmaht-dac%20yaml.load&type=code), I don't see any uses within a repo where dcicutils is used (where the version of pyyaml it uses will increments due to this dcicutils change).

Reply:
I built a cgap-portal(branch: dcicutils-with-pyyaml-6-and-sheet-utils | version: 14.2.0.1b1) with this dcicutils/pyyaml update and deployed to cgap-supertest and it seems fine (login/logout, create access-key, and ran submit-metadata-bundle).